buildSrc/mac.gradle

```diff
@@ -28,6 +28,11 @@ ext.MAC = [:]
 MAC.canBuild = IS_MAC && IS_64
 if (!MAC.canBuild) return;
 
+def TARGET_ARCH = "x86_64"
+if (hasProperty('targetArch')) {
```

Comment:
Normally, our external gradle properties are all upper-case, so `TARGET_ARCH` might be a better name for the property and `targetArch` a better name for the local variable.